docs/customization.md

```diff
@@ -93,63 +93,57 @@ If you publish your artifact to a distant maven repository, do not forget to add
 
 You may not want to generate *all* models in your project. Likewise, you may want just one or two apis to be written.  If that's the case, you can use system properties or [global properties](./global-properties.md) to control the output.
 
-The default is generate *everything* supported by the specific library.  Once you enable a feature, it will restrict the contents generated:
+The default is generate *everything* supported by the specific library. Once you enable a feature, it will restrict the contents generated:
 
 ```sh
 # generate only models
-java -Dmodels {opts}
+--global-property models
```

Comment:
@wing328 this charge isn't correct. The syntax in the README is system properties, and value-less properties like models/APIs can only be passed as system properties like exist in the README.

To change to our CLI args, any occurrence must follow the `generate` CLI command and must have key=value syntax.

The reason our `-D` option was removed from CLI is because people mistakenly thought this worked in any position and modified system properties similar to the `mvn` app. Even I was confused by this when I began contributing.

Reply:
@jimschubert Let's not advertise the usage of `-D` (any position), shall we? The main reason is what you've pointed out - confusing.